open-logic-config.sty

```diff
@@ -411,7 +411,7 @@
 \DeclareDocumentCommand \varAssign { m m m o } {
     \IfNoValueTF {#4}
         % optional argument not present
-        { #1 \sim{#3} #2 }
+        { #1 \sim_{#3} #2 }
         % optional argument present
         { #1 = #2^{#3}_{#4} }
```

Comment:
Not that it matters for the OLP where the optional argument isn't used but don't you want _x and ^o?

Reply:
Thanks! The place where I've seen this (Sider's Logic for Philosophers I think) uses this presentation (variable on top, assigned object at the bottom). Perhaps a better default is the `s[o/x]` notation you're considering in #251 ? I'll put that in.